cmake/developer_package/plugins/plugins.cmake

```diff
@@ -3,6 +3,7 @@
 #
 
 include(CMakeParseArguments)
+include(${CMAKE_SOURCE_DIR}/cmake/developer_package/clang_tidy/clang_tidy.cmake)
```

Comment:
use relative path, because Developer package can be used as standalone project w/o OpenVINO